packages/web/components/ad-banner/ad-banner-content.tsx

```diff
@@ -29,6 +29,7 @@ export const AdBannerContent: React.FC<AdBannerContentProps> = memo(
         target="_blank"
         rel="noopener noreferrer"
         href={external_url}
+        data-testid="ad-banner-link"
```

Comment:
Usually, relying on test IDs is not recommended as it may indicate that the item is not properly accessible. I believe `screen.getByRole("link", { name: subheader })` should work. If not, let's add aria-label instead